templates/default/sql/dml/1.4.0.sql.erb

```diff
@@ -5,4 +5,5 @@ REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("jupyter_group", "<
 -- the path is hardcoded because it is the path inside the docker image, not on the machine.
 REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("anaconda_dir", "/srv/hops/anaconda");
 REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("docker_registry_name", "<%= @registry_host %>");
-REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("docker_registry_port", "<%= node['hops']['docker']['registry']['port'] %>");
\ No newline at end of file
+REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("docker_registry_port", "<%= node['hops']['docker']['registry']['port'] %>");
+DELETE FROM `hopsworks`.`variables` WHERE `id`='provided_python_lib_names';
```

Comment:
why did you remove this one?

Reply:
It is not needed anymore. Now we list all libraries in the environment so this variable is useless.